source/Map.svelte

```diff
@@ -506,9 +508,15 @@
       }
     });
   });
+
+  const closeSettingsForSmallDevices = () => {
+    if (innerWidth <= 1100) {
+      areSettingsShown.update((previous) => !previous);
+    }
+  };
 </script>
 
-<div bind:this={mapElement} id="map"></div>
+<div on:click={closeSettingsForSmallDevices} bind:this={mapElement} id="map" />
```

Comment:
```suggestion
<div on:pointerdown={closeSettingsForSmallDevices} bind:this={mapElement} id="map" />
```

So it works when you zoom or pan the map on a touch device.